src/index.js

```diff
@@ -1,6 +1,5 @@
 const throat = require('throat');
-const pify = require('pify');
-const workerFarm = require('worker-farm');
+const Worker = require('jest-worker').default;
```

Comment:
`.default` is a babel implementation detail, and should never be exposed in an entry point. is there a bug filed on `jest-worker` for this?

Reply:
I don't see any.

Reply:
Would you mind filing one and linking to it here?

Reply:
I'm not very good with words, but I don't mind.  What should I say in this issue?

Reply:
Whats up with this patch. It seems good to me. I don't see an issue with the .default thing. In fact, its how https://github.com/rogeliog/create-jest-runner/blob/master/lib/createJestRunner.js#L1 does it, which was recommended in issue #23

Reply:
I suppose it doesn't have to block this issue; but it's indeed a huge problem for babel interop details to pollute the ecosystem.

I've filed https://github.com/facebook/jest/issues/5803